src/node/api_clients/rpc.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import json
+from dataclasses import asdict
+from src.node.api_clients.base_client import BaseClient
+
+logger = logging.getLogger(__name__)
+
+
+class RPC(BaseClient):
```

Comment:
I would not spend too much time on JSON RPC because we want to deprecate it in favour of REST API.

Reply:
I've left it for comparison purposes. This way we can check that the REST API works the same as the old RPC protocol.